src/plugins/discover/public/application/angular/doc_table/create_doc_table_react.tsx

```diff
@@ -87,8 +87,6 @@ export function convertDirectiveToRenderFn(
 
     return () => {
       if (!rejected) {
-        // for cleanup
```

Comment:
This change is now in 3 PRs. Let's rebase this PR after #3847 is merged.

Reply:
done